website/profile/views.py

```diff
@@ -730,3 +730,23 @@ def unserialize_schools(auth, **kwargs):
     verify_user_match(auth, **kwargs)
     unserialize_contents('schools', unserialize_school, auth)
     # TODO: Add return value
+
+
+@must_be_logged_in
+def request_export(auth):
+    mails.send_mail(
+        to_addr=settings.SUPPORT_EMAIL,
+        mail=mails.REQUEST_EXPORT,
+        user=auth.user,
+    )
+    return ''
```

Comment:
Views should always return some feedback to the client about what action was taken. Even something like: 

```
{'message': 'Sent account export request'}
```